UOP1_Project/Assets/Scripts/StateMachineTest/ChaseDataObject.cs

```diff
@@ -0,0 +1,10 @@
+﻿using UnityEngine;
+
+[CreateAssetMenu(fileName = "ChaseData", menuName = "State Machines/Tests/Data/Chase Data")]
+public class ChaseDataObject : ScriptableObject
```

Comment:
Similar consideration to the other comment about the "Scriptable" prefix. Here `ChaseDataObject` should really be `ChaseDataSO` for clarity on what type of object it is?